### PR TITLE
+ zh-HK main page, replacing zh-CN copy

### PR DESCRIPTION
**Pull request by JeromeMartinez**

### CODE REVIEW

**Comment by XhmikosR** [comment]
And the menu?

**Comment by JeromeMartinez** [comment]
The menu is modified in _.html

**Comment by XhmikosR** [comment]
No, I mean, should you remove zh-HK from the menu too? Or this is just a duplicate file?

**Comment by JeromeMartinez** [comment]
I don't see the issue.
Previous zh-HK/_.html was duplicated content from zh-CH/_.html, replaced with this commit by real zh-HK. Nothing removed, just replaced.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,68 @@
...
+# MediaArea-Website
+
+[![Build Status](https://travis-ci.org/MediaArea/MediaArea-Website.svg?branch=master)](https://travis-ci.org/MediaArea/MediaArea-Website)
+
+# How to install
+
+## Dependencies
+
+* Apache web server >= 2.2 (should also work on nginx and lighttpd)
+    * mod rewrite is recommended
```
Comment (JeromeMartinez) [inline README.md]: mod\_rewrite?

**Pull request merged**